gxa/src/main/java/uk/ac/ebi/atlas/controllers/rest/DASFeaturesControllerNew.java

```diff
@@ -65,7 +49,8 @@ public DASFeaturesControllerNew(ExperimentTrader experimentTrader,
     @ResponseStatus(value = HttpStatus.INTERNAL_SERVER_ERROR)
     public ModelAndView InternalServerHandleException(Exception e) {
         ModelAndView mav = new ModelAndView("error-page");
-        mav.addObject("exceptionMessage", e.getMessage());
+        mav.addObject("exceptionMessage", "Unexpected error occurred, email us if this presists.");
+        LOGGER.error("Error getting Differential json results.",e.getMessage());
```

Comment:
Include the full `e` in the error. You saw how masking the stack trace gave us trouble in debugging the Reactome HTTPS error. #LearningTheHardWay